ESP8266/ESP8266.cpp

```diff
@@ -832,10 +840,29 @@ void ESP8266::_oob_watchdog_reset()
         _sock_i[i].open = false;
     }
 
-    _conn_status = NSAPI_STATUS_DISCONNECTED;
+    // Makes possible to reinitialize
+    _conn_status = NSAPI_STATUS_ERROR_UNSUPPORTED;
     _conn_stat_cb();
 }
 
+void ESP8266::_oob_busy()
+{
+    char status[5];
+    if (_parser.recv("%4[^\"]\n", status)) {
+        if (strcmp(status, " s...\n") == 0) {
+            ; //TODO maybe do something here, or not...
+        } else if (strcmp(status, "p...\n") == 0) {
+            ; //TODO maybe do something here, or not...
+        } else {
+            MBED_ERROR(MBED_MAKE_ERROR(MBED_MODULE_DRIVER, MBED_ERROR_CODE_EBADMSG), \
+                    "ESP8266::_oob_busy: busy doing what?\n");
+        }
+    } else {
+            MBED_ERROR(MBED_MAKE_ERROR(MBED_MODULE_DRIVER, MBED_ERROR_CODE_EBADMSG), \
+                    "ESP8266::_oob_busy: busy doing what?\n");
```

Comment:
I don't see the value of this error message. Messages should be targeted to users. So don't ask users what the ESP is doing.
Instead report that ESP is reporting busy status.

Reply:
Could change to something like "unrecognized AT busy state"